tests/tt_eager/python_api_testing/conv/conv_unit_test_utils.py

```diff
@@ -50,7 +50,7 @@ def create_conv_weight_tensor(torch_tensor, K, C, R, S, in1_block_h, in1_block_w
     B_ = ttnn.Tensor(torch.flatten(torch_tensor).tolist(), weights_shape, ttnn.bfloat16, ttnn.ROW_MAJOR_LAYOUT).pad(
         weights_channels_padded_shape, (0, 0, 0, 0), 0.0
     )
-    B_tiled_host = ttnn.experimental.tensor.convert_conv_weight_tensor_to_tiled_layout(B_, in1_block_h, in1_block_w)
+    B_tiled_host = ttnn.tensor.convert_conv_weight_tensor_to_tiled_layout(B_, in1_block_h, in1_block_w)
     return B_tiled_host
```

Comment:
@mywoodstock , do you have an opinion on this?
Should this be:
ttnn.conv.convert_conv_weight_tensor_to_special_padding_tiled_layout
or
ttnn.convert_conv_weight_tensor_to_special_padding_tiled_layout

Reply:
I don't have a preference, but im thinking `conv.convert...` makes more sense since its conv specific helper function.

Reply:
I have moved them from tensor module to conv2d module @mywoodstock @eyonland